appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.mapper.js

```diff
@@ -59,11 +61,92 @@ function generateTableRows(items, isReview = false) {
 		// TODO: Use updated date function when ready - dateISOStringToDisplayDate
 		{ text: dateToDisplayDate(new Date(comment.created)) },
 		{
-			html: `<a href="/comments/${comment.id}/${isReview ? 'review' : 'view'}">${
+			html: `<a href="./interested-party-comments/${comment.id}/${isReview ? 'review' : 'view'}">${
```

Comment:
should this be a relative URL? I don't think we're using them elsewhere, so just checking